api-channel/src/main/java/com/synopsys/integration/alert/api/channel/convert/SimpleMessageConverter.java

```diff
@@ -23,9 +23,11 @@ public SimpleMessageConverter(ChannelMessageFormatter messageFormatter) {
     }
 
     @Override
-    public List<String> convertToFormattedMessageChunks(SimpleMessage simpleMessage) {
+    public List<String> convertToFormattedMessageChunks(SimpleMessage simpleMessage, String jobName) {
         ChunkedStringBuilder chunkedStringBuilder = new ChunkedStringBuilder(messageFormatter.getMaxMessageLength());
 
+        String nonBreakingSpace = messageFormatter.getNonBreakingSpace();
+        appendSection(chunkedStringBuilder, String.format("Job%sname:%s%s", nonBreakingSpace, nonBreakingSpace, jobName));
```

Comment:
Main change

Reply:
Bold?

Reply:
I'll go through the classes and bold the entire statement